frontend/components/TableContainer/DataTable/TruncatedTextCell/TruncatedTextCell.tsx

```diff
@@ -55,7 +55,11 @@ const TruncatedTextCell = ({
         clickable
         delayHide={200} // need delay set to hover using clickable
       >
-        {value}
+        <>
+          {value}
+          <div className="safari-hack">&nbsp;</div>
+          {/* Safari hack work around */}
```

Comment:
can we add a bit more description on what the hack is. just so if someone needs to work in this area, they can get some context on why we had to do this.